transport/src/androidUnitTest/kotlin/transport/core/StateMachineTest.kt

```diff
@@ -287,7 +287,7 @@ class StateMachineTest {
     }
 
     @Test
-    fun verifyFullStateTransitionCycle() {
+    fun `verify full state transition cycle`() {
```

Comment:
Just wondering why this test doesn't follow the same naming structure of "when...". Is it on purpose?